### PR TITLE
docs: (Core) add dropdown for switching to old docs versions

### LINKED ISSUE #3107: Displaying documentation for older versions of the library
Our documentation page is only showing the current version of the library and there's no option for the user to access older versions.

### PR DESCRIPTION
**Pull request by mikerodonnell89**
fixes #3107 

The menu is very long and should have a max-height - this will be fixed as part of https://github.com/SAP/fundamental-ngx/issues/3522

### CODE REVIEW

**Comment by InnaAtanasova** [comment]
Hey @mikerodonnell89 once you select an older version you no longer have the option to select another version (the dropdown menu is not available anymore). Is this the expected behaviour?

**Comment by mikerodonnell89** [comment]
> Hey @mikerodonnell89 once you select an older version you no longer have the option to select another version (the dropdown menu is not available anymore). Is this the expected behaviour?

Yes, I can't go back and edit the functionality of those older versions, but moving forward we can keep the list up to date so future versions will have the option of going back to this version and later

File: apps/docs/src/app/documentation/core-helpers/toolbar/toolbar.component.ts [hunk 1]
```diff
@@ -69,9 +74,29 @@ export class ToolbarDocsComponent implements OnInit {
 
     ngOnInit(): void {
         this.cssUrl = this.sanitizer.bypassSecurityTrustResourceUrl('assets/sap_fiori_3.css');
+
```
Comment (droshev) [inline apps/docs/src/app/documentation/core-helpers/toolbar/toolbar.component.ts:77]: Is there a way to automate this?

**Review by JKMarkowski — COMMENTED** [review]
Let's spend some time doing research, if adding new version during release can be automated, besides that code LGTM

**Comment by mikerodonnell89** [comment]
Automation will be handled with this issue https://github.com/SAP/fundamental-ngx/issues/3577

**Review by InnaAtanasova — APPROVED** [review]
Further investigation how to automate it will be done as a separate task. As discussed today this PR can be merged.

**Pull request merged**